readme.txt

```diff
@@ -2,6 +2,8 @@
 
 Contributors:      google
 Tested up to:      5.7
+Requires at least: 5.3
+Requires PHP:      5.6
```

Comment:
As the issue description says, we don't need `Requires PHP`

Reply:
Is there any issue in adding this as well? Does it hurting having this value there as well, just to be overly careful.

Reply:
No issue except that it's unnecessary.

Contrary to `Requires at least`, which is currently still needed but being phased out, this header is no longer needed anywhere and can be removed.

That's why I want to only bring back `Requires at least` for the time being.